packages/react-sdk/package.json

```diff
@@ -29,6 +29,8 @@
     "access": "public"
   },
   "dependencies": {
+    "@optimizely/js-sdk-logging": "^0.1.0",
+    "@optimizely/optimizely-sdk": "3.2.0",
```

Comment:
Can we maybe have it receive patch updates? Unless there is a strong case for pinning it down to the patch version.